app_dart/test/request_handlers/update_agent_health_test.dart

```diff
@@ -46,6 +48,8 @@ void main() {
       expect(agent.agentId, 'test');
       expect(response.agent.isHealthy, true);
       expect(agent.healthDetails, 'bar detail');
+      expect(tabledataResourceApi.rows, 1);
```

Comment:
Is it possible to use `list` on this line?

Reply:
Changed to use list. But not sure the superiority over directly using 'rows'?

Reply:
"Prefer to test via the public interface of a component." https://abseil.io/tips/135